<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tabwidgetConfermaPrenotazioneColazioneInCamera</class>
 <widget class="QTabWidget" name="tabwidgetConfermaPrenotazioneColazioneInCamera">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>623</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Struttura Alberghiera: Conferma Prenota Colazione In camera per giorno successivo</string>
  </property>
  <widget class="QWidget" name="tab">
   <attribute name="title">
    <string>Conferma Prenotazione</string>
   </attribute>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="labelIntroduzione">
      <property name="text">
       <string>Prima di confermare pu√≤ far decidere al cliente quante volte prenotare il piatto scelto in precedenza per gli altri clienti della camera:</string>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="btnConfermaPrenotazione">
      <property name="text">
       <string>Conferma Prenotazione</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="btnIndietro">
      <property name="text">
       <string>Indietro</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QGroupBox" name="groupboxPreferenzeColazioneInCameraCliente">
      <property name="title">
       <string>Preferenze scelte dal cliente per la colazione in camera:</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupboxSalatoConfermaPrenotazioneColazioneInCamera">
         <property name="title">
          <string>Salato:</string>
         </property>
         <layout class="QFormLayout" name="formLayoutSalato"/>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupboxDolceConfermaPrenotazioneColazioneInCamera">
         <property name="title">
          <string>Dolce:</string>
         </property>
         <layout class="QFormLayout" name="formLayoutDolce"/>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QGroupBox" name="groupboxBevandeConfermaPrenotazioneColazioneInCamera">
         <property name="title">
          <string>Bevande:</string>
         </property>
         <layout class="QFormLayout" name="formLayoutBevande"/>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
